lib/subtitles.ex

```diff
@@ -1,3 +1,14 @@
 defmodule Subtitles do
+  def get_format(subtitle) do
+    subtitle = subtitle |> normalize_line_endings()
 
+    subtitle |> is_vtt?() && :vtt ||
+    subtitle |> is_srt?() && :srt ||
+    :unknown
+  end
+
+  defp is_vtt?(subtitle), do: subtitle |> String.starts_with?("WEBVTT")
+  defp is_srt?(subtitle), do: Regex.match?(~r/^\n?1\n/, subtitle)
```

Comment:
If a file just containing a 1 gets presented to this lib it will tell you that it's a srt file. Which is incorrect. It needs to be able to parse at least one subtitle queue of the type srt to make sure that it is in fact an srt.

Reply:
We are currently not building a subtitle format validator, this is currently meant to be a simple pointer to differentiate between srt- and vtt-files. If needed later, this can be fleshed out with a more robust implementation and a full validator.